x/sequencer/ante/ante.go

```diff
@@ -24,16 +34,21 @@ func NewAnteHandler(options ante.HandlerOptions) (sdk.AnteHandler, error) {
 
 	standardSigVerificationDecorator := ante.NewSigVerificationDecorator(options.AccountKeeper, options.SignModeHandler)
 
+	standardDeductFeeDecorator := ante.NewDeductFeeDecorator(options.AccountKeeper, options.BankKeeper, options.FeegrantKeeper, options.TxFeeChecker)
```

Comment:
out of the scope of this PR, but:
<img width="723" alt="image" src="https://github.com/warp-contracts/sequencer/assets/84311757/a55b21a5-1512-4cdb-91c2-c238a734a26d">

Reply:
I changed to functions from `cosmossdk.io/errors` module.